ui/app/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -59,11 +57,11 @@ export default class SettingsTab extends PureComponent {
         </div>
         <div className="settings-page__content-item">
           <div className="settings-page__content-item-col">
-            <SimpleDropdown
-              placeholder={t('selectCurrency')}
```

Comment:
We could add placeholder support to `Dropdown` :thinking: I don't really care much about it being removed though.

Reply:
Oh, wait, was this _never used_ because the `selectedOption` was always set? Hah.